polylogue/automation.py

```
@@ -0,0 +1,183 @@
+from __future__ import annotations
+
+import sys
+import json
+from dataclasses import dataclass, field
+from importlib import resources
+from pathlib import Path
+from textwrap import dedent
+from typing import Dict, Iterable, List, Optional
```

Ruff (0.15.9)

[warning] Import from `collections.abc` instead: `Iterable`

Import from `collections.abc`

(UP035)